/tf_odom_to_base_link:
  ros__parameters:
    /child_frame_id: base_link
    /frame_id: odom
    /rotation/w: 1.0
    /rotation/x: 0.0
    /rotation/y: 0.0
    /rotation/z: 0.0
    /translation/x: 0.0
    /translation/y: 0.0
    /translation/z: -0.41
    use_sim_time: false
